samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -31,10 +31,18 @@ static void attention_blink(struct k_work *work)
 {
 	static int idx;
 	const uint8_t pattern[] = {
-		BIT(0) | BIT(1),
-		BIT(1) | BIT(2),
-		BIT(2) | BIT(3),
-		BIT(3) | BIT(0),
+#if DT_NODE_EXISTS(DT_ALIAS(led0))
```

Comment:
Do we need to add this to the "bt_mesh_elem_elements[]" too?

Reply:
I think we are good. The LED used by the Light CTRL server is a pwm LED. lc_pwm_led.c will give compile error if pwm-led0 is not avaviable for the device.